File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        n = len(temperatures)
        res = []


        for x in range(n):
            count = 0
            for y in range(x+1,n):
                if( temperatures[y] > temperatures[x] ):
                    count = y-x
                    break
            res.append(count)
        return res

File: Data Structures & Algorithms/k-closest-points-to-origin/submission-0.cs
public class Solution {
    public int[][] KClosest(int[][] points, int k) 
    {
    PriorityQueue <int[], int> maxHeap = new PriorityQueue <int[], int>();

    foreach (var point in points)
    {
        int dist = point[0] * point[0] + point[1] * point[1];
        maxHeap.Enqueue(point, -dist);
        if(maxHeap.Count  > k)
        {
            maxHeap.Dequeue();
        }
    }
    var res = new List<int[]>(); 
    while (maxHeap.Count > 0){
    res.Add(maxHeap.Dequeue());
    }
        return res.ToArray();
    }
}
